src/Service.Tests/CosmosTests/MutationTests.cs

```diff
@@ -242,6 +243,49 @@ public async Task MutationMissingRequiredPartitionKeyValueReturnError()
             Assert.AreEqual("The argument `_partitionKeyValue` is required.", response[0].GetProperty("message").ToString());
         }
 
+        /// <summary>
+        /// Mutation can be performed on the authorized fields
+        /// </summary>
+        [TestMethod]
+        public async Task CanCreateItemWithAuthorizedFields()
+        {
+            // Run mutation Add planet;
+            string id = Guid.NewGuid().ToString();
+            string mutation = $@"
+mutation {{
+    createEarth (item: {{ id: ""{id}"" }}) {{
+        id
+    }}
+}}";
+            JsonElement response = await ExecuteGraphQLRequestAsync("createEarth", mutation, variables: new());
+
+            // Validate results
+            Assert.AreEqual(id, response.GetProperty("id").GetString());
+        }
+
+        /// <summary>
+        /// Mutation performed on the unauthorized fields throws permission denied error
```

Comment:
```suggestion
        /// Mutation performed on the unauthorized fields throws permission denied error because the
        /// field `name` is an excluded field for the create operation on the anonymous role defined
        /// for entity 'earth'
```

Reply:
Updated